integration_tests/test_coverage.py

```diff
@@ -87,7 +89,6 @@ def _get_current_coverage(coverage_config):
         )[0])
 
     # Remove coverage related directories.
-    shutil.rmtree(kcov_ouput_dir, ignore_errors=True)
```

Comment:
Why are we not removing this anymore?

Reply:
According to my understanding, this line of code removes the kcov_output directory, which contains the coverage information (results) generated by kcov. In your posted issue #6 , it is asked to not remove this directory. I think probably we can fix that issue here...

Reply:
Let's have a separate commit for that. We want to keep it only for `profile --devel`. Here we are keeping it on every profile which is not the desired behavior because when calling this via the CI we do want to clean up all the files before exiting the test. In `profile --devel` this would be useful for contributors to see what lines are not covered.

Reply:
Sure. I have reverted the code. Thanks!